release_notes.md

```diff
@@ -1,18 +1,57 @@
 # What's New
 
-## Enhancement
+## Enhancements
 
-- Upon **Alert** or **Incident** created or updated with an MITRE Technique ID, MITRE modules *Group*, *Tactic*, *Technique*, *Sub-Technique*, *Software*, and *Mitigation* gets correlated.
-- Introduced new dashboard **MITRE ATT&CK Matrices** which display MITRE Matrices.
-- Introduced new playbook **Get MITRE ATT&CK Data** to get MITRE ATT&CK data manually.
+- MITRE modules *Group*, *Tactic*, *Technique*, *Sub-Technique*, *Software*, and *Mitigation* are now correlated with an alert or incident on updating the alert or incident with a MITRE Technique ID
+- A new dashboard **MITRE ATT&CK Matrices** displays MITRE matrices
+- A new **Get MITRE ATT&CK Data** playbook gets MITRE ATT&CK data when executed
 
 ## Bug Fixes
 
-- Changed *Group* and *Tactic* field type from *One To One* to *Many To Many* in **Alert** and **Incident**.
+- Changed *Group* and *Tactic* field type from *One To One* to *Many To Many* in **Alert** and **Incident** modules
 
-## Depricated
+## Deprecations
 
-- Removed **MITRE Admin** role as **Full App Permission** contains required permission for MITRE modules. On upgrade, it will not remove the existing **MITRE Admin** role.
-- Removed *Hunt* module permission from **Full App Permission** as **SOAR Framework** minimum version `v1.1.0` solution pack contains required permission for *Hunt* module. On upgrade, it will not remove the existing permission from **Full App Permission**.
+- Removed the role **MITRE Admin** as **Full App Permission** contains the necessary permissions for working with MITRE modules.
 
-> **Note**: On updating `Technique ID` in Alert or Incident, new MITRE data will be appended in correlations. 
\ No newline at end of file
+    > **NOTE**: After an upgrade existing permissions in the **MITRE Admin** role remain unaffected.
+
+- Removed *Hunt* module permissions from **Full App Permission** as **SOAR Framework** `v1.1.0` and later contains necessary permissions for the *Hunt* module.
+
+    > **NOTE**: After an upgrade existing permissions in the **Full App Permission** role remain unaffected.
+
+## Known Issues
```

Comment:
This need to be added in usage.md. As it is not the part of this release. It would be affected in upcoming version as well.